data/load-seed-data.js

```diff
@@ -51,10 +51,10 @@ async function run() {
         });
 
         return client.query(`
-                    INSERT INTO legends (name, special_type_id, special_group, type_1, type_2, other_forms, owner_id)
-                    VALUES ($1, $2, $3, $4, $5, $6, $7);
+                    INSERT INTO legends (name, special_type_id, special_group, type_1, type_2, other_forms, image_url, description, owner_id)
```

Comment:
great work implementing the join, and adding some more additional logic (bc who doesn't like seeing an image of a pokemon).  Nice work!